### DISCUSSION TITLE
Issues in Update to 20230117

### DISCUSSION THREAD

**Discussion by digixmax** [rern/rAudio#1196] [Bug Reports]
I updated my RPi3 from build 20230101 to 20230117 after which Spotify stopped working -- I found out that its renderer radio button got reset to off. I was not able to reenable & reconfigure it, so I flashed back to 20221117 to make another attempt to get to 20230117, but there is no auto-update path from 20221117 to 20230117, and there is not yet any download link for 20230117.

**Comment by nsummy** [comment]
I just did my first ever rAudio install today and had the same problem.  Spent half the afternoon messing with it.  Seems like there is an "expiration date" foolishly hardcoded into the system where it only allows updates for a certain time before it requires a new image.  Clearly this presents a problem if a new image doesn't get release within this period.  The fix was to just go into this file and edit it to a later date.  That allowed me to update.  Though I'm not sure its wise considering the mpd service won't start for me now :)

https://github.com/rern/rAudio-1/discussions/620

**Comment by digixmax** [comment]
I should also add that I was not able to "restore" from a backup.gz file created with 20230101 to fix the hosed up renderer setting/state -- rAudio became unresponsive when the restore script completed.
